### PR TITLE
namesgenerator: new adjective, two inspiring Computing Science authors

### PR DESCRIPTION
**Pull request by Russtopia**
Signed-off-by: Russ Magee <rmagee@gmail.com>

**- What I did**
Added a new adjective to the namesgenerator and two authors who inspired myself and many other young computer scientists in the 1980s and 1990s.

**- How I did it**
Edits to namegenerator.go - little code impact

**- How to verify it**
Examine diffs and verify spelling and grammar; ensure passes Jenkins test suites

**- Description for the changelog**
namesgenerator: New adjective 'recursing'; addition of A.K. Dewdney and Douglas R. Hofstadter

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -257,6 +258,9 @@ var (
 		// Bailey Whitfield Diffie - American cryptographer and one of the pioneers of public-key cryptography. https://en.wikipedia.org/wiki/Whitfield_Diffie
 		"diffie",
 
+		// A. K. (Alexander Keewatin) Dewdney, Canadian mathematician, computer scientist, author and filmmaker. Contributor to Scientific American's "Computer Recreations" from 1984 to 1991. Author of Core War (program), The Planiverse, The Armchair Universe, The Magic Machine, The New Turing Omnibus, and more. https://en.wikipedia.org/wiki/Alexander_Dewdney
+		"dewdney",
```
Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Could you replace one of these new names with a female? See this comment at the top of the file; https://github.com/moby/moby/blob/9d61eee75a3681fdff7f0c7e3232f4ccd3dfcaf6/pkg/namesgenerator/names-generator.go#L106

**Review by thaJeztah — CHANGES_REQUESTED** [review]
Thanks for contributing, and sorry for the delay in reviewing; I left a comment 👍

File: pkg/namesgenerator/names-generator.go (hunk 1 above)
Comment (Russtopia) [reply pkg/namesgenerator/names-generator.go]: Sure! I'll prep a new patch. Actually I can think of a few to add.

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -136,6 +136,9 @@ var (
 		// Stefan Banach - Polish mathematician, was one of the founders of modern functional analysis. https://en.wikipedia.org/wiki/Stefan_Banach
 		"banach",
 
+		// Buckaroo Banzai and his mentor Dr. Hikita perfectd the "oscillation overthruster", a device that allows one to pass through solid matter. - https://en.wikipedia.org/wiki/The_Adventures_of_Buckaroo_Banzai_Across_the_8th_Dimension
```
Comment (Russtopia) [inline pkg/namesgenerator/names-generator.go]: Iff we can make BTTF references (line 193), why not BB? :)

File: pkg/namesgenerator/names-generator.go [hunk 3]
```diff
@@ -252,15 +258,15 @@ var (
...
+		// A. K. (Alexander Keewatin) Dewdney, Canadian mathematician, computer scientist, author and filmmaker. Contributor to Scientific American's "Computer Recreations" from 1984 to 1991. Author of Core War (program), The Planiverse, The Armchair Universe, The Magic Machine, The New Turing Omnibus, and more. https://en.wikipedia.org/wiki/Alexander_Dewdney
+		"dewdney",
+
 		// Satish Dhawan - Indian mathematician and aerospace engineer, known for leading the successful and indigenous development of the Indian space programme. https://en.wikipedia.org/wiki/Satish_Dhawan
 		"dhawan",
 
 		// Bailey Whitfield Diffie - American cryptographer and one of the pioneers of public-key cryptography. https://en.wikipedia.org/wiki/Whitfield_Diffie
 		"diffie",
 
-		// A. K. (Alexander Keewatin) Dewdney, Canadian mathematician, computer scientist, author and filmmaker. Contributor to Scientific American's "Computer Recreations" from 1984 to 1991. Author of Core War (program), The Planiverse, The Armchair Universe, The Magic Machine, The New Turing Omnibus, and more. https://en.wikipedia.org/wiki/Alexander_Dewdney
```
Comment (Russtopia) [inline pkg/namesgenerator/names-generator.go]: Move for alphabetization

File: pkg/namesgenerator/names-generator.go (hunk 2 above)
Comment (Russtopia) [reply pkg/namesgenerator/names-generator.go]: My diff adds Vera Rubin's entry at line 651 but it doesn't show up in the pull request diff. Strange.

**Review by thaJeztah — APPROVED** [review]
LGTM :smile:

**Comment by Russtopia** [comment]
Hooray! Thank you everyone.

**Pull request merged**